sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultStep.java

```diff
@@ -143,16 +140,6 @@ public Status getStatus() {
         }
     }
 
-    @Override
-    public List<Element<?>> getChildren() {
-        return Collections.emptyList();
-    }
-
-    @Override
```

Comment:
Interrupting a step implies halting the thread of execution within that step, which we haven't ever supported. In practice when you interrupt a plan it results in the current step finishing its operation and then the following step sitting in an interrupted (`WAITING`?) state